installer/conf/kube.conf

```diff
@@ -119,8 +125,8 @@
       max_retry_wait 9m
      </match>
 
-     <match oms.api.ContainerNodeInventory**>
-      type out_oms_api
+     <match oms.containerinsights.ContainerNodeInventory**>
```

Comment:
I still need to merge ci_feature into feature/customMetrics. That is why this change is seen. I merged from ci_feature into my topic branch